src/modules/StandardActions.ts

```diff
@@ -0,0 +1,110 @@
+import { BigNumber } from 'bignumber.js';
+import { Operation } from './operate/Operation';
+import { Contracts } from '../lib/Contracts';
+import {
+  TxResult,
+  MarketId,
+  address,
+  AmountReference,
+  AmountDenomination,
+  ContractCallOptions,
+  ConfirmationType,
+} from '../types';
+
+export class StandardActions {
+  private operation: Operation;
+  private contracts: Contracts;
+
+  constructor(
+    operation: Operation,
+    contracts: Contracts,
+  ) {
+    this.operation = operation;
+    this.contracts = contracts;
+  }
+
+  public async deposit({
+    accountOwner,
+    marketId,
+    amount,
+    accountNumber = new BigNumber(0),
+    options,
+  }: {
+    accountOwner: address,
+    marketId: BigNumber | string,
+    amount: BigNumber | string,
+    accountNumber: BigNumber | string,
+    options?: ContractCallOptions,
+  }): Promise<TxResult> {
+    const isEth = new BigNumber(marketId).eq(MarketId.ETH);
+    const operation = this.operation.initiate({
+      usePayableProxy: isEth,
```

Comment:
use ProxyType instead (usePayableProxy: true) is deprecated. Same comment for withdraw